src/test/java/ca/team2706/frc/robot/commands/drivebase/DriverAssistVisionTest.java

```diff
@@ -0,0 +1,130 @@
+package ca.team2706.frc.robot.commands.drivebase;
+
+import ca.team2706.frc.robot.config.Config;
+import ca.team2706.frc.robot.subsystems.DriveBase;
+
+import jaci.pathfinder.Pathfinder;
+import jaci.pathfinder.Trajectory;
+
+import mockit.Expectations;
+import mockit.Mocked;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class DriverAssistVisionTest {
+
+    private final DriverAssistVision driverAssistVision = new DriverAssistVision();
+
+    // Since these two classes are accessed statically (they are singletons), we have to mock 
+    // all instances and stub out the static initializer
+
+    @Mocked(stubOutClassInitialization = true)
+    private DriveBase driveBase;
+
+    @Mocked(stubOutClassInitialization = true)
+    private Config config;
```

Comment:
We don't want to be mocking config, since the classes being tested actually use it.

Reply:
Config is now "partially mocked" as described in Section 13 of http://jmockit.github.io/tutorial.html. The reason for this partial mock is that expected results for the tests depend on  
the values of certain parameters in Config (e.g. location of camera on robot) and those values could have to be tweaked during testing or right before a competition. We don't want to have to change the tests when we're tweaking parameters, and the specific values of the parameters (and associated expected results) don't affect what the test if trying to achieve. If you have a preferred different approach, let us know.